soc/xtensa/intel_adsp/cavs/include/intel_tgl_adsp/adsp_imr_layout.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2023 Intel Corporation
```

Comment:
- IMR with capitals (Isolated Memory Region)
- "to load image from IMR memory"

Reply:
ok tried to make the commit message better...